Syntax.in

```diff
@@ -0,0 +1,31 @@
+Syntax: 
+
+program = "{" compound_stm "}"
+
+compound_stm = stm[stm]
```

Comment:
Based on you definition, a compund_stm is a single statement or two statements at most. Maybe stmt { stmt } fits better?